### ISSUE TITLE
HOLD #48400 [$250] Red dot (RBR) doesn't go away even though this split expense has been paid/settled ("paid elsewhere") by all parties

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#48423]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**
**Reproducible in staging?:** Needs Reproduction
**Reproducible in production?:** Needs Reproduction
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @jamesdeanexpensify 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1724971827199829

## Action Performed:
1. Create a split expense in a group chat
2. Settle the expense (Paid elsewhere) by all the members the expense was split 
## Expected Result:
No RBR in LHN
## Actual Result:
RBR displayed in LHN and unable to dismiss it.
## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

![image](https://github.com/user-attachments/assets/a2949b40-7f85-4405-944a-5068582a0be0)

Onyx Data:

```
{
    "chatType": "group",
    "lastActorAccountID": "1520269",
    "lastMessageText": "Yup",
    "lastVisibleActionCreated": "2024-08-28 18:45:43.504",
    "ownerAccountID": 0,
    "participants": {
        "1381556": {
            "hidden": false,
            "notificationPreference": "always",
            "role": "member"
        },
        "1520269": {
            "hidden": false,
            "notificationPreference": "always",
            "role": "member"
        },
        "4037741": {
            "hidden": false,
            "notificationPreference": "always",
            "role": "admin"
        }
    },
    "policyID": "_FAKE_",
    "reportID": "4421636436784885",
    "reportName": "",
    "stateNum": 0,
    "statusNum": 0,
    "type": "chat",
    "isEdited": true,
    "lastMessageHtml": "Yup",
    "currency": "USD",
    "description": "",
    "errorFields": {},
    "hasOutstandingChildRequest": false,
    "hasOutstandingChildTask": false,
    "isCancelledIOU": false,
    "isOwnPolicyExpenseChat": false,
    "isPinned": true,
    "isWaitingOnBankAccount": false,
    "lastActionType": "ADDCOMMENT",
    "lastReadSequenceNumber": 0,
    "lastReadTime": "2024-08-28 18:45:43.504",
    "lastVisibleActionLastModified": "2024-08-28 18:45:43.504",
    "nonReimbursableTotal": 0,
    "notificationPreference": "always",
    "oldPolicyName": "",
    "permissions": [
        "read",
        "write",
        "share"
    ],
    "private_isArchived": "",
    "total": 0,
    "unheldTotal": 0,
    "welcomeMessage": "",
    "writeCapability": "all",
    "lastMessageTranslationKey": "",
    "maxSequenceNumber": 0,
    "isLoadingPrivateNotes": false,
    "privateNotes": {
        "1520269": {
            "note": ""
        }
    },
    "managerID": 0
}
```

Add any screenshot/video evidence

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021833077301118863367</li>
        <li>Upwork Job ID: 1833077301118863367</li>
        <li>Last Price Increase: 2025-02-17</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @ikevin127</details>

**Comment by MelvinBot** [comment]
This has been labelled "Needs Reproduction". Follow the steps here: https://stackoverflowteams.com/c/expensify/questions/16989

**Comment by muttmuure** [comment]
tomorrow

**Comment by muttmuure** [comment]
I think it's likely this needs to go into #newdot-quality.

**Comment by Vicolas11** [comment]
Hello, has this issue been assigned to any Upwork freelancer?

**Comment by ikevin127** [comment]
> Hello, has this issue been assigned to any Upwork freelancer?

Not yet, looking for proposals (if the issue is reproducible).

**Comment by ikevin127** [comment]
💰 Looking for proposals (if the issue is reproducible).

**Comment by ikevin127** [comment]
💰 Looking for proposals (if the issue is reproducible).

**Comment by ikevin127** [comment]
- Not overdue, there's been multiple reports of a similar issue: https://github.com/Expensify/App/issues/48400 and https://github.com/Expensify/App/issues/48397 (which is on HOLD until the first one is addressed).

@muttmuure I'd suggest to put this one on [HOLD #48400] until that one is fixed then we can re-test and look for proposals if the PR for the first one did not fix this issue.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by ikevin127** [comment]
Still on hold.

**Comment by ikevin127** [comment]
Not overdue, on hold.

**Comment by ikevin127** [comment]
Coming from the HOLD issue https://github.com/Expensify/App/issues/48400#issuecomment-2390337662:

> From this https://github.com/Expensify/App/issues/48400#issuecomment-2367773018, seems part of the behavior is expected. Closing it out for now, but we can re-open if it's reported again as an issue

@muttmuure I think we're good to close out this one based on the same logic / reasoning mentioned in https://github.com/Expensify/App/issues/48400#issuecomment-2367773018.

**Comment by ikevin127** [comment]
We had a similar issue here which was closed as NAB:
- https://github.com/Expensify/App/issues/48400

the consensus there was that it is expected to still have RBR show because of existing violations, see https://github.com/Expensify/App/issues/48400#issuecomment-2367773018:

> - The red circle on showing on the report preview is working as intended, because the report has violations and we show that no matter what the state of the report is.
> - After paying, the red circle should remain in the report preview, for the same reason as in the first bullet
> - After clearing the cache, the report preview isn't showing the red circle, and after clicking on the preview, the transaction previews aren't showing the red circle. These are bugs because they should be showing for the same reason as in the first bullet

The https://github.com/Expensify/App/issues/48400#issuecomment-2390337662 that the issue was closed with:

> From this https://github.com/Expensify/App/issues/48400#issuecomment-2367773018, seems part of the behavior is expected. Closing it out for now, but we can re-open if it's reported again as an issue

This seems to be a dupe of that so I think we're good to close it as NAB.

cc @muttmuure

**Comment by ikevin127** [comment]
♻️ **Update:** We're expecting this issue to be fixed by the solution proposed in issue https://github.com/Expensify/App/issues/52037, we're working on opening the PR as the contributor was just assigned. I will get back with a retest request for this issue once that PR is merged and deployed.

**Comment by ikevin127** [comment]
PR https://github.com/Expensify/App/pull/52988 from issue https://github.com/Expensify/App/issues/52037 was merged. We should re-test this to see whether that PR fixed this as well and if it did - we can close as fixed.

**Comment by ikevin127** [comment]
PR https://github.com/Expensify/App/pull/52988 from issue https://github.com/Expensify/App/issues/52037 was merged. We should re-test this to see whether that PR fixed this as well and if it did - we can close as fixed.

**Comment by ikevin127** [comment]
PR https://github.com/Expensify/App/pull/52988 from issue https://github.com/Expensify/App/issues/52037 was merged. We should re-test this to see whether that PR fixed this as well and if it did - we can close as fixed.

**Comment by ikevin127** [comment]
@muttmuure PR https://github.com/Expensify/App/pull/52988 from issue https://github.com/Expensify/App/issues/52037 was merged. We should re-test this to see whether that PR fixed this as well and if it did - we can close as fixed.

**Comment by ikevin127** [comment]
@muttmuure PR https://github.com/Expensify/App/pull/52988 from issue https://github.com/Expensify/App/issues/52037 was merged. We should ask for a re-test to determine whether that PR fixed this as well and if it did - we can close as fixed.

**Comment by ikevin127** [comment]
@muttmuure PR https://github.com/Expensify/App/pull/52988 from issue https://github.com/Expensify/App/issues/52037 was merged. We should ask for a re-test to determine whether that PR fixed this as well and if it did - we can close as fixed.

**Comment by ikevin127** [comment]
@lschurr PR https://github.com/Expensify/App/pull/52988 from issue https://github.com/Expensify/App/issues/52037 was merged. We should ask for a re-test to determine whether that PR fixed this as well and if it did - we can close as fixed.

**Comment by lschurr** [comment]
Asked for a retest here: https://expensify.slack.com/archives/C9YU7BX5M/p1739982152385889

**Comment by izarutskaya** [comment]
Issue is not reproducible 
MacBook Air 14.1 Chrome

https://github.com/user-attachments/assets/686fb29b-f2a6-41ca-99ea-987b84a5583c

**Comment by lschurr** [comment]
Closing!
